lib/cisco_node_utils/vpc.rb

```diff
@@ -62,6 +66,14 @@ def enable
       config_set('vpc', 'feature', state: '')
     end
 
+    def fabricpath_feature
```

Comment:
Even better, move them into the Feature class. I know, I know, beyond the scope of this story... :-)